### DISCUSSION TITLE
Web site & CV

### DISCUSSION THREAD

**Discussion by mdekstrand** [rsms/inter#633] [Show and tell]
Earlier this year, I revamped my integrated academic web site and CV to use Inter (paired with Space Grotesk for headings), and today updated to Inter 4.0: <https://md.ekstrandom.net/>

**Comment by IngrownMink4** [comment]
I really like your website! And you can make the dark mode even better by choosing a better color (#b0d7ff) for the URLs and adding the `color-scheme: light dark` property to style the search-box.

This is how it looks now:
![a website using inter (actual)](https://github.com/user-attachments/assets/732fbe5e-646c-41a1-9d55-51b9a83c9693)

And this is how can be improved with these changes:
![a website using inter (APCA applied)](https://github.com/user-attachments/assets/0058ee6b-f3fb-454a-b750-70623bb9564a)

**Reply by mdekstrand** [reply]
Thank you! I've applied the search color theme setting (didn't know about that!), will think about the colors (I'm currently using [Flexoki](https://stephango.com/flexoki) as-is, and want to give more thought before tweaking that).

**Reply by IngrownMink4** [reply]
> Thank you! I've applied the search color theme setting (didn't know about that!), will think about the colors (I'm currently using [Flexoki](https://stephango.com/flexoki) as-is, and want to give more thought before tweaking that).

Understandable :)
I suggested the color change to make it [APCA](https://github.com/Myndex/SAPC-APCA/blob/master/documentation/APCA_in_a_Nutshell.md) compliant, because right now it's not, which means that some people might have trouble reading the text. However, I'm not sure if there are any other suitable color variants that match the Flexoki theme, though.

![apca-failed-example](https://github.com/user-attachments/assets/ffdf07cd-72eb-4ce9-b182-79cc54542041)

**Reply by mdekstrand** [reply]
That's a good point, thanks.

The light theme passes APCA, but dark clearly doesn't.
